packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -451,6 +461,23 @@ export default decorate([
               </div>
             </fieldset>
           )}
+          {type !== 's3' && (
+            <fieldset className='form-group form-group'>
+              <div className='input-group form-group'>
+                <span className='align-middle'>
+                  {_('remoteUseVhdDirectory')}
+                  <Tooltip content={_('remoteUseVhdDirectoryTooltip')}>
+                    <Icon icon='info' size='lg' />
+                  </Tooltip>
+                </span>
+                <Toggle
+                  className='align-middle pull-right'
+                  onChange={effects.setUseVhdDirectory}
+                  value={useVhdDirectory === true}
+                />
```

Comment:
Shouldn't it be disabled if `type === 's3'`?

Reply:
this fieldset in only shown on remote that are not s3